testsuite/MDAnalysisTests/coordinates/test_gro.py

```diff
@@ -418,6 +419,57 @@ def test_growriter_resid_truncation():
     # larger digits should get truncated
     assert line.startswith('56789UNK')
 
+class TestGrowriterReindex():
+    def __init__(self):
```

Comment:
We don't use init methods for tests.  If you want to reuse `self.u` make it into a pytest fixture